pkg/querier/queryrange/queryrangebase/roundtrip.go

```diff
@@ -66,7 +66,7 @@ func (cfg *Config) Validate() error {
 	}
 
 	if len(cfg.ShardAggregations) > 0 && !cfg.ShardedQueries {
-		return errors.New("shard_aggregation requires parallelise_shardable_queries=true")
+		return errors.New("shard_aggregations requires parallelise_shardable_queries=true")
```

Comment:
Is this error still reflective of the correct value required?

Reply:
Yes.